bindings/bindingtester/spec/bindingApiTester.md

```diff
@@ -47,44 +47,44 @@ instructions:
 Data Operations
 ---------------
 
-PUSH <item>
+#### PUSH &lt;item&gt;
 
     Pushes the provided item onto the stack.
```

Comment:
As it renders now, all of the descriptions show up as code literals (because they are indented). IMO, I think it looks kind of cool, but was that the intention?

Reply:
The intention was to keep the descriptions as code literals. The alternative would be to un-indent the descriptions and make sure to put any variables in the descriptions between backticks so they show up as code. Do you have a preference? (I couldn't decide, so I left everything indented)

Reply:
I mean, I personally think it looks fine and would be willing to merge. I'm not sure if anyone else has any opinion.